src/components/Pagination.tsx

```diff
@@ -0,0 +1,72 @@
+import { Dispatch, SetStateAction } from "react";
+import { IconButton } from "src/components";
+import { Css, Palette } from "src/Css";
+import { SelectField } from "src/inputs";
+import { useTestIds } from "src/utils";
+
+export type PageSettings = {
+  page: number;
+  perPage: number;
+};
+
+type PaginationProps = {
+  label: string;
+  hasNextPage: boolean;
```

Comment:
Thought: We could probably drop `hasNextPage` and infer it from `PageSettings.page` and `totalCount / perPage`, per other comment to add `totalCount` to remove the hard-coded drop down.

Reply:
@allangaldinosilva / @roberthdg I guess in retrospect we just don't need `totalCount`? That's cool, so n/m I guess?

Reply:
We do, actually we will need it for the new requirement that @roberthdg told, we are gonna show the current page info close to the navigation buttons.

![image](https://user-images.githubusercontent.com/108748822/219786191-d16a03a1-1672-4bd1-926c-2fea9fa69fb6.png)

I pushed these changes in the last commit, when you get a chance to re-review this I would appreciate.